site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -96,116 +92,96 @@ links.forEach(link => link.addEventListener('click', updatePage));
 For example, you might deal with success by hiding a previously displayed progress indicator, like this:
 
 ```js
-appHistory.addEventListener('navigatesuccess', event => {
+navigation.addEventListener('navigatesuccess', event => {
   loadingIndicator.hidden = true;
 });
 ```
 
 Or you might show an error message on failure (i.e., if the `Promise` passed to `transitionWhile` rejected):
 
 ```js
-appHistory.addEventListener('navigateerror', event => {
+navigation.addEventListener('navigateerror', event => {
   loadingIndicator.hidden = true; // also hide indicator
   showMessage(`Failed to load page: ${event.message}`);
 });
 ```
 
-The "navigateerror" event handler, which receives an `ErrorEvent`, is particularly handy as it's guaranteed to receive any errors from your code that's setting up a new page.
+The "navigateerror" event listener, which receives an `ErrorEvent`, is particularly handy as it's guaranteed to receive any errors from your code that's setting up a new page.
 You can simply `await fetch()` knowing that if the network is unavailable, the error will eventually be routed to "navigateerror".
 
-### Transition Context and Rollback
-
-During the period between the initial "navigate" event and the exact time these terminal events fire, there'll also be an object available that gives some context on the navigation that's occurring.
-It's available at `appHistory.transition`.
-You're able to use this object to _rollback_ this navigation, pretending as if the navigation never occurred.
-Your site might choose to do this in response to a user action, or perhaps if the loaded URL is invalid and should never appear in the user's history.
-For example:
-
-```js
-appHistory.addEventListener('navigateerror', event => {
-  const attemptedLoadURL = location.href;
-  const previousURL = appHistory.transition.from;
-  console.warn('Failed to load', attemptedLoadURL, 'returning to', previousURL);
-  appHistory.transition.rollback();
-});
-```
-
-Like many of the features in the App History API, this call to `rollback()` is a semantic action that implies meaning to the browser.
-If a clicked `<a href>` fails to load and rolls back, the URL won't appear in the user's session history (for Forward or Back navigation).
-
-{% Aside %}
-
-Although calling `rollback()` on error is useful in some situations, you should avoid it in the general case.
-You're more likely to show an error message, perhaps so the user can try a reload to get their content.
-
-{% endAside %}
-
 ### Abort Signals
 
 Since you're able to do asynchronous work while preparing a new page, it's possible that the transition your code is handling (to load a specific URL or state) might get preempted, or considered out-of-date.
-This might happen because the `rollback()` method was invoked, the user just clicked on another link, or your code itself throws an error.
+This might happen because the user just clicked on another link, or your code itself throws an error.
```

Comment:
I'm actually not sure "your code itself throws an error" is something that can cause this... maybe "your code itself performs another navigation"?

Reply:
huh, yeah. Was that link always wrong, or was it a change to the spec?

Reply:
I think that was always wrong, although maybe it was referring to how a transition is considered a failure if the IIAFE throws an error. (Failure is different than preemption, but this is the closest I can think of.)